card/lib/cardio.rb

```diff
@@ -50,26 +48,19 @@ def set_config config
   
     def set_paths paths
       @@paths = paths
-      paths.add 'tmp/lib'
-      paths.add 'tmp/set'
-      paths.add 'tmp/set_pattern'
-      paths.add 'db/migrate_deck_cards', :with=>'db/migrate_cards'
+      add_gem_path 'tmp/set', :root => root
+      add_gem_path 'tmp/set_pattern', :root => root
```

Comment:
It seems the only use cases for the option are referring to the deck root.  I'm sure there were other cases as one point, but no longer.  Is it worth making the add_path call the same in Cardio and Wagn?  In the latter, there is a paths argument, but not in the former.  I have an idea of how to do that even though it is called in the paths method.